src/cpu/gemm/gemm.cpp

```diff
@@ -192,6 +203,22 @@ dnnl_status_t gemm_s8x8s32(const char *transa, const char *transb,
                 B, LDB, bo, beta, C, LDC, co, false);
 #endif
 
+#if DNNL_PPC64
```

Comment:
Kindly suggest to follow the `elif` practice as in headers.

```suggestion
#elif DNNL_PPC64
```

Reply:
Done